WordPress/Classes/Extensions/UITextField+TextHelper.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+extension UITextField
+{
+    func nonNilTrimmedText() -> String {
+        guard let str = text?.trim() else {
+            return ""
+        }
+        return str
+    }
```

Comment:
On a second thought I think I agree with you. I'm not sure why `UITextField.text` is an optional, and adding to the problem, the header says the default is `nil` and the documentation says the default is `""`. It's probably good to coalesce the empty cases to "".

I'm not sure if I'd add trimming to the mix, but the function name is clear enough about it :+1: